example_brem_bkpl.pro

```diff
@@ -0,0 +1,52 @@
+Pro example_brem_bkpl
+    
+    ; Example script to use brem_nontherm.pro for bremsstrahlung calculation: 
+    ; calculate x-ray spectrum from a broken-power-law electron distribution   
+  
+    eph = findgen(1000)        ; Photon energies
+    emin = 10.                 ; Min electron energy
+    emax = 2.e3                ; Max electron energy
+    ebrk = 100.                ; break energy
+    bin= 0.01                  ; Electron energy bin (in log space)
+    nbin = fix((alog(emax)-alog(emin))/bin)
+    eel = findgen(nbin+1)*bin + alog(emin)
+    eel = get_edges( exp(eel), /mean )		;Electron energies
+    n_0 = 1.e10                ; Ambient density
+    delta1 = 3.5               ; Electron spectral index below ebrk
+    delta2 = 8.                ; Electron spectral index above ebrk
+    
+    ; Compute electron density distribution
+    brm2_distrn, eel, delta1, delta2, emin, ebrk, emax, e_dist
+    n_e = 1.e5*e_dist
```

Comment:
What does this do?  Is this a normalization?  If so, it may be best to include the normalization factor in the list of parameters above that the user can set.  (And tell them how the normalization is parametrized, since everybody chooses a different way to normalize their distributions.)